### DISCUSSION TITLE
Problem with Choco internalization

### DISCUSSION THREAD

**Discussion by oscarcipriano2000** [chocolatey/choco#3370] [General] [answered]
Good morning,

I am experiencing a small problem. I am trying to internalize a package that was taken from the community (Ex. Notepad++) but I get to a point where it asks me to enter powershell commands that are not recognized: (Install-ChocolateyZipPackage)

May I know if there is a need for some kind of license or is the "free" plan to internalize the packages ok?

Thank you!

**Comment by pauby** [comment, marked as answer]
Internalization is a feature of Chocolatey for Business. If you have a Chocolatey for Business license, please run `choco support` for the Support options available and you can then reach out for help.

If you are using Chocolatey CLI open-source, then the automatic internalization is not available.
